core/src/main/resources/hudson/PluginManager/sidepanel.jelly

```diff
@@ -0,0 +1,117 @@
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<!--
+  Sidepanel component for Plugins Manager
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+  <st:documentation>
+    <st:attribute name="page">
+      Adds the highlighted class to the current page item in the list
+    </st:attribute>
+  </st:documentation>
+
+  <div id="tasks" class="app-plugin-manager__sidebar">
+    <div class="jenkins-app-bar">
+      <div class="jenkins-app-bar__content">
+        <h1>${%Plugins}</h1>
+      </div>
+    </div>
+
+    <div class="task">
+      <span class="task-link-wrapper">
+        <a href="./" class="task-link ${page == 'updates' ? 'task-link--active' : ''}">
+          <div class="task-icon-link">
+            <l:icon src="symbol-download" />
+          </div>
+          <span class="task-icon-text">
+            ${%Updates}
+          </span>
+          <j:if test="${app.updateCenter.updates.size() != 0}">
+            <span class="task-icon-badge">
+              ${app.updateCenter.updates.size()}
+            </span>
+          </j:if>
+        </a>
+      </span>
+    </div>
+
+    <div class="task">
+      <span class="task-link-wrapper">
+        <a href="./available" class="task-link ${page == 'available' ? 'task-link--active' : ''}">
+          <div class="task-icon-link">
+            <l:icon src="symbol-shopping-bag" />
+          </div>
+          <span class="task-icon-text">
+            ${%Available plugins}
+          </span>
+        </a>
+      </span>
+    </div>
+
+    <div class="task">
+      <span class="task-link-wrapper">
+        <a href="./installed" class="task-link ${page == 'installed' ? 'task-link--active' : ''}">
+          <div class="task-icon-link">
+            <l:icon src="symbol-plugins" />
+          </div>
+          <span class="task-icon-text">
+            ${%Installed plugins}
+          </span>
+        </a>
+      </span>
+    </div>
+
+    <div class="task">
+      <span class="task-link-wrapper">
+        <a href="./advanced" class="task-link ${page == 'advanced' ? 'task-link--active' : ''}">
+          <div class="task-icon-link">
+            <l:icon src="symbol-settings" />
+          </div>
+          <span class="task-icon-text">
+            ${%Advanced settings}
+          </span>
+        </a>
+      </span>
+    </div>
+
```

Comment:
Shouldn't there be a Back button as well?

Reply:
Ideally the `Manage Jenkins` breadcrumb should be resurrected so we don't need that.

i.e https://github.com/jenkinsci/jenkins/pull/6126 (not sure why it was closed, if there was something missing in it, cc @daniel-beck

Reply:
Yes, it is part of the breadcrumb then it is not needed indeed.

Reply:
> not sure why it was closed, if there was something missing in it

https://github.com/jenkinsci/jenkins/pull/6485#issuecomment-1175251633